src/components/simulation/Additional_Resources.tsx

```diff
@@ -86,8 +86,8 @@ const Additional_Resources: React.FC<{ setStage: setStageType }> = ({
         className="ta-center"
         style={{ fontSize: '1.2em', marginBottom: '1rem' }}
       >
-        Let's look at some additional resources to help you learn more about
-        personal and business finance
+        Let's spend the next 5 minutes looking at some additional resources to help you learn more about
```

Comment:
Open to suggestions of this language, and if anyone has ideas on what is most clear for the user.

Reply:
I think this is great 🙌